cf_units/__init__.py

```diff
@@ -1867,9 +1867,13 @@ def convert(self, value, other, ctype=FLOAT64):
             64-bit double-precision (cf_units.FLOAT64) used for conversion
             when `value` is not a NumPy array or is a NumPy array composed of
             NumPy integers. The default is 64-bit double-precision conversion.
+        * inplace (bool):
+            If ``False``, return a deep copy of the value array. If ``True``,
+            convert the values in-place. A new array will be created if
+            ``value`` is an integer NumPy array.
 
         Returns:
-            float or numpy.ndarray of appropriate float type.
+            float or numpy.ndarray of appropriate float type
```

Comment:
I'll add it back. I don't use them in docstrings like this, so I removed it out of habit...